openapi-v1.yaml

```diff
@@ -2628,6 +2628,71 @@ definitions:
       - array:read-write
       - array:admin
 
+  GroupAsset:
+    description: an asset of a group. Can be array notebook ml_model or udf
+    type: object
+    properties:
+      name:
```

Comment:
You need type too

Reply:
The urls are of the form `arrays/groups noteboos/groups` so the type is implied.